src/intern.rs

```diff
@@ -24,6 +24,10 @@ where
         &self.rev_strings[data]
     }
 
+    pub(crate) fn untern_vec(&self, data: &Vec<TargetType>) -> Vec<&str> {
```

Comment:
https://stackoverflow.com/q/40006219/155423

Reply:
Thanks! I really appreciate these comments!

Changed data to be a slice of `TargetType`.